### PR TITLE
(fix) : Improved PageSpeedInsight Performance,  "dev.to" username is always in lowercase, Repositories count in 'Base Content' & follow-up plugin issues and pull requests count

### PR DESCRIPTION
**Pull request by JayantGoel001**
Hello @lowlighter,

image

## Improved the PageSpeedInsight Performance by reducing the number of API calls.

Initially, **4** API calls were made for each category 
`https://www.googleapis.com/pagespeedonline/v5/runPagespeed?category=${category}&url=https://www.google.com&key=XXXXXX`

Updated, Only **1** call is made for all categories
`https://www.googleapis.com/pagespeedonline/v5/runPagespeed?url=https://www.google.com&category=performance&category=accessibility&category=best-practices&category=seo&key=XXXXXX`

## (fix) : Repositories count in 'Base Content'
Initially --- > 

Updated --->
image

## followup plugin issues and pull requests count

Initially --->
image

but in my issue tab

image

in my pull request tab
- merged 
  image
- unmerged(closed)
  image

updated --->

image

The change I made was to add `archived: false` in every query of the user

### CODE REVIEW

File: source/plugins/base/queries/user.x.graphql [hunk 1]
```diff
@@ -30,7 +30,7 @@ query BaseUserX {
     repositoriesContributedTo(includeUserRepositories: true) {
       totalCount
     }
-    repositories(last: 0) {
+    repositories(last: 0, ownerAffiliations : OWNER) {
```
Comment (lowlighter) [inline source/plugins/base/queries/user.x.graphql:33]: I think this should use `$affiliations` to use the constraint set by [`repositories_affiliations`](https://github.com/lowlighter/metrics/tree/master/source/plugins/base#repositories_affiliations)

Comment (JayantGoel001) [reply source/plugins/base/queries/user.x.graphql:33]: Actually When I used `$affliations` then I was able to retrieve only my repositories but not other users. So using `$affliations` will create issue in metrics(web) but `$ownerAffiliations` is able to give us the repositories count of every user.

File: source/plugins/followup/queries/user.graphql [hunk 2]
```diff
@@ -1,26 +1,26 @@
 query FollowupUser {
-  issues_open:search(query: "is:issue author:$login is:open", type: ISSUE, first: 0) {
+  issues_open:search(query: "is:issue author:$login is:open archived:false", type: ISSUE, first: 0) {
```
[the remark this answers is not in the document]
Comment (JayantGoel001) [reply source/plugins/followup/queries/user.graphql:2]: No Actually 😅,
This is sort of GitHub's default configuration.

When we click on issue or pr tab `$archived : false` is always there.
image

[the remark this answers is not in the document]
Comment (JayantGoel001) [reply source/plugins/followup/queries/user.graphql:2]: Yes, Toggle Option will be better.🤔

File: source/plugins/base/queries/user.x.graphql (hunk 1 above)
Comment (JayantGoel001) [reply source/plugins/base/queries/user.x.graphql]: done😁

File: source/plugins/followup/queries/user.graphql (hunk 2 above)
[the remark this answers is not in the document]
Comment (JayantGoel001) [reply source/plugins/followup/queries/user.graphql]: done😁

**Comment by JayantGoel001** [comment]
Hello @lowlighter,
PWA Support has been added to PageSpeed

> PWA Not Installed
not

> PWA Installed
Untitled

File: source/plugins/base/queries/field.repositories.graphql [hunk 3]
```diff
@@ -1,6 +1,6 @@
-query BaseFieldRepositories {
+query BaseFieldRepositories{
   $account(login: "$login") {
-    repositories(last: 0) {
+    repositories(last: 0, $affiliations) {
```
Comment (lowlighter) [inline source/plugins/base/queries/field.repositories.graphql:3]:
```suggestion
    repositories(last: 0 $affiliations) {
```
The `affiliations` already contains a comma (`,`) so no need to add one in the query (this is to support empty affiliations)

(same for other occurences)

File: source/plugins/pagespeed/examples.yml [hunk 4]
```diff
@@ -30,3 +30,13 @@
...
+- name: Succint report with PWA
+  uses: lowlighter/metrics@latest
+  with:
+    filename: metrics.plugin.pagespeed.svg
+    token: NOT_NEEDED
+    base: ""
+    plugin_pagespeed: yes
+    plugin_pagespeed_token: ${{ secrets.PAGESPEED_TOKEN }}
+    plugin_pagespeed_url: https://lecoq.io
+    plugin_pagespeed_pwa: yes
```
Comment (lowlighter) [inline source/plugins/pagespeed/examples.yml:42]: To pass this test, you need to patch [`tests/mocks/api/axios/get/pagespeed.mjs`](https://github.com/lowlighter/metrics/blob/master/tests/mocks/api/axios/get/pagespeed.mjs) to integrate mocked data for PWA and make the builds pass

File: source/plugins/base/queries/field.repositories.graphql (hunk 3 above)
Comment (JayantGoel001) [reply source/plugins/base/queries/field.repositories.graphql]: done 😁

**Comment by JayantGoel001** [comment]
> Still some changes to make to pass the builds and overall looks good 👍
> 
> Also, if you want to add new features/patchs, it's better to make separate pull requests, it makes merging and reviewing easier 😅

Ok I will make sure of that next time🙂

File: source/plugins/pagespeed/examples.yml (hunk 4 above)
Comment (JayantGoel001) [reply source/plugins/pagespeed/examples.yml:42]: done 😁

**Review by lowlighter — APPROVED** [review]

**Comment by lowlighter** [comment]
Thanks a lot for your contribution @JayantGoel001 !

**Pull request merged**